x/ecocredit/orderbook/orderbook.go

```diff
@@ -0,0 +1,70 @@
+package orderbook
+
+import (
+	"context"
+
+	"github.com/cosmos/cosmos-sdk/orm/model/ormdb"
+	marketplacev1beta1 "github.com/regen-network/regen-ledger/api/regen/ecocredit/marketplace/v1beta1"
+	orderbookv1beta1 "github.com/regen-network/regen-ledger/api/regen/ecocredit/orderbook/v1beta1"
+	ecocreditv1beta1 "github.com/regen-network/regen-ledger/api/regen/ecocredit/v1beta1"
```

Comment:
how about removing `v1beta1` from import alias?

Reply:
this is done automatically by buf and is desired: https://docs.buf.build/generate/managed-mode#default